README.md

```diff
@@ -30,6 +30,9 @@ dependencies {
 
     // Optionally, you can include the widgets library for ScaleBar, etc.
     implementation 'com.google.maps.android:maps-compose-widgets:2.15.0'
+
+    // Android Maps Utils KTX
+    implementation 'com.google.maps.android:maps-utils-ktx:4.0.0'
```

Comment:
Re-rasiing - we don't need to tell developers to list maps-utils-ktx as a dependency anymore since maps-compose-utils pulls that in now per gradle/libs.versions.toml.